go/host/p2p/p2p.go

```diff
@@ -162,9 +184,22 @@ func (p *p2pImpl) broadcast(msg Message) error {
 	return nil
 }
 
+// Sends a message to the sequencer.
+func (p *p2pImpl) sendToSequencer(msg message) error {
+	msgEncoded, err := rlp.EncodeToBytes(msg)
+	if err != nil {
+		return fmt.Errorf("could not encode message to send to sequencer. Cause: %w", err)
+	}
+	// TODO - #718 - Use better method to identify sequencer?
```

Comment:
Thoughts?

Reply:
I think it's fine for now, maybe set it up as like `p.currSequencer` as a slight abstraction to make it clearer what's going on. Longer term when we're decentralized I wonder if it would be better to use any peers to do the batches catch-up rather than all relying the current leader.